mmtrack/models/sot/siamrpn.py

```diff
@@ -245,6 +247,138 @@ def track(self, img, bbox, z_feat, avg_channel):
+            A1 = torch.norm(quadrilateral[0:2] - quadrilateral[2:4]) * \
+                torch.norm(quadrilateral[2:4] - quadrilateral[4:6])
+            A2 = (x2 - x1) * (y2 - y1)
+            s = torch.sqrt(A1 / A2)
+            w = s * (x2 - x1)
+            h = s * (y2 - y1)
+            bbox = torch.stack((cx, cy, w, h))
+        elif length == 4:
+            bbox = bbox_xyxy_to_cxcywh(quadrilateral)
+        else:
+            NotImplementedError(f'The length of quadrilateral: {length} is \
+                 not supported')
+        return bbox
+
+    def _bbox_cxcywh_to_xywh(self, bbox):
+        """Convert bbox format from [cx, cy, w, h] to [x, y, w, h]"""
+        assert len(bbox) == 4
+        bbox[0] = max(bbox[0] - bbox[2] / 2, 0)
+        bbox[1] = max(bbox[1] - bbox[3] / 2, 0)
+        return bbox
+
+    def _bbox_xyxy_to_xywh(self, bbox):
+        """Convert bbox format from [x1, y1, x2, y2] to [x, y, w, h]"""
+        assert len(bbox) == 4
+        bbox[2] = max(bbox[2] - bbox[0], 0)
+        bbox[3] = max(bbox[3] - bbox[1], 0)
+        return bbox
+
+    def simple_test_vot(self, img, frame_id, gt_bboxes, image_shape=None):
+        """Test using VOT criteria.
+
+        Args:
+            img (Tensor): of shape (1, C, H, W) encoding input image.
+            frame_id (int): the id of current frame in the video.
+            gt_bboxes (list[Tensor]): list of ground truth bboxes for each
+                image with shape (1, 4) in [tl_x, tl_y, br_x, br_y] format.
+            image_shape (tuple | list): used as the bounds when calculating
+                overlap.
+
+        Returns:
+            bbox_pred (Tensor): in [tl_x, tl_y, br_x, br_y] format.
+            best_score (Tensor): the tracking bbox confidence in range [0,1],
+                and the score of initial frame is -1.
+        """
+        if frame_id == 0:
+            self.init_frame_id = 0
+        if self.init_frame_id == frame_id:
+            # initialization state
+            gt_bboxes = gt_bboxes[0][0]
+            self.memo = Dict()
+            self.memo.bbox = self._get_axis_aligned_bbox(gt_bboxes)
+            self.memo.z_feat, self.memo.avg_channel = self.init(
+                img, self.memo.bbox)
+            bbox_pred = img.new_tensor([1.])
+            best_score = -1.
+        elif self.init_frame_id > frame_id:
+            # unknown state, namely the skipping frame after failure
+            bbox_pred = img.new_tensor([0.])
+            best_score = -1.
+        else:
+            # normal tracking state
+            best_score, self.memo.bbox = self.track(img, self.memo.bbox,
+                                                    self.memo.z_feat,
+                                                    self.memo.avg_channel)
+            # self.memo.bbox = torch.round(self.memo.bbox)
+            # convert bbox to region
+            track_bbox = self.memo.bbox.cpu().numpy()
+            track_bbox = self._bbox_cxcywh_to_xywh(track_bbox)
```

Comment:
first transform the format of bbox, then convert it to numpy and use bbox2region